.tekton/rapidast-llm-pull-request.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/redHatProductSecurity/rapidast?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/cancel-in-progress: "true"
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
+      == "development"
+  labels:
+    appstudio.openshift.io/application: rapidast
+    appstudio.openshift.io/component: rapidast-llm
+    pipelines.appstudio.openshift.io/type: build
+  name: rapidast-llm-on-pull-request
+  namespace: secaut-tenant
+spec:
+  pipelineRef:
+    name: rapidast-pipeline
+  params:
+  - name: git-url
+    value: '{{source_url}}'
+  - name: revision
+    value: '{{revision}}'
+  - name: output-image
+    value: quay.io/redhatproductsecurity/rapidast-llm:on-pr-{{revision}}
+  - name: image-expires-after
+    value: 7d
+  - name: dockerfile
+    value: containerize/Containerfile.garak
+  - name: skip-checks
+    value: "true"
+  - name: prefetch-input
```

Comment:
Since prefetching is disabled in the `Containerfile`, this parameter should be removed to also disable prefetching with cachi2 in the pipeline. This will prevent the prefetching of dependencies

Reply:
added a new commit to remove it